doc/api.apib

```diff
@@ -678,14 +678,16 @@ Workflow Instance belonging to a Backfill entity.
 
 Resources related to inspect detailed status of Workflow Instances.
 
-## Active States [/{version}/status/activeStates{?component,workflow}]
+## Active States [/{version}/status/activeStates{?component,workflow,components}]
 
 + Parameters
     + version: `v3` (enum[string]) - API version
         + Members
             + `v3`
     + component: `styx-canary` (optional, string) - Component ID
     + workflow: `StyxCanary`  (optional, string) - Workflow ID - requires Component ID to also be specified
+    + components: `styx-canary,styx-canary1,styx-canary3` (optional, string) - Component IDs, IDs should be comma
+    separated.
```

Comment:
Let's also make it clear that if `components` is provided, the other two parameters will be ignored